htsget-search/Cargo.toml

```diff
@@ -42,7 +42,7 @@ aws-sdk-s3 = { version = "0.28", optional = true }
 aws-config = { version = "0.55", optional = true }
 
 # Url storage
-hyper-rustls = { version = "0.24", features = ["rustls-native-certs", "http2", "http1"], optional = true }
```

Comment:
Downgrade?

Reply:
Yes, this is a bit annoying, but actix-web uses rustls 0.20, which is not the most recent version. So the common code in `htsget-config` `tls.rs` can only use rustls 0.20 in order to be compatible with the actix-web server setup. I might ask on the actix-web devs to see if they are planning to upgrade to rustls 0.21, although I feel like this might be a breaking change, so it may not be upgraded until actix-web 5.0, which could be a while away.